Tests/SourceKitLSPTests/LocalClangTests.swift

```diff
@@ -181,15 +181,20 @@ final class LocalClangTests: XCTestCase {
   }
 
   func testCodeAction() async throws {
-    guard let ws = try await staticSourceKitTibsWorkspace(name: "CodeActionCxx") else { return }
-    if ToolchainRegistry.shared.default?.clangd == nil { return }
-
-    let loc = ws.testLoc("SwitchColor")
-    let endLoc = ws.testLoc("SwitchColor:end")
+    let testClient = try await TestSourceKitLSPClient()
```

Comment:
What does clangd do with arguments? I assume we'd end up having to create a `compile_commands.json` here at some point if we ended up wanting to have more C/C++ related tests.

Reply:
I don’t know but I assume it just does the equivalent of `clang /path/to/file.c`.

We can always use a `MultiFileTestWorkspace` to define a compilation DB. And if that becomes common, we can always consider adding a subclass for file + compilation DB similar to how we have `SwiftPMTestWorkspace`.